sympy/physics/units/dimensions.py

```diff
@@ -572,3 +572,87 @@ def is_consistent(self):
         # dimensions
         # in vector language: the set of vectors do not form a basis
         return self.inv_can_transf_matrix.is_square
+
+    def buckingham_pi_theorem(self, list_of_derived_quantities):
+        """
+        list_of_derived_quantities is the input of derived quantities in the form of a list of tuples (quantity_name, dims) where quantity_name
+        is the name of the physical quantity and dims is an instance of Dimension class which possesses the dimensions of the quantity. This function
+        is designed to take in a set of physical variables (all essentially dimensionful) and returns one set of all possible exponents,
+        as guided by the Buckingham's pi theorem (raise the list of quantities to the list of exponents to get the dimensionless numbers).
+
+        One of the next functions get_dimensionless_numbers() is used to produce the requisite result in the form of a list of instances of Dimension
+        class signifying the actual dimensionless quantities.
+
+        For example, consider an RLC circuit with resistance ```$R$```, inductance ```$L$``` and capacitance ```$C$``` connected to a constant
+        voltage source ```$V_{amp}$```, where current ```$I(t)$``` is given by the equation (https://en.wikipedia.org/wiki/RLC_circuit).
```

Comment:
Sorry for my previous message, I think it works with just one backtick (I erroneously wrote three earlier). Have you tried running `make html` in `sympy/doc`? It should create the HTML documentation so you can see whether the LaTeX expressions are rendered correctly.